django/core/management/commands/inspectdb.py

```diff
@@ -153,7 +157,7 @@ def table2model(table_name):
                         '' if '.' in field_type else 'models.',
                         field_type,
                     )
-                    if field_type.startswith('ForeignKey('):
+                    if field_type.startswith('ForeignKey(') or field_type.startswith('OneToOneField('):
```

Comment:
`startswith()` accepts tuple, so we can use
```python
if field_type.startswith(('ForeignKey(', 'OneToOneField(')):
```

Reply:
Ack, will change.